tests/python/test_moe.py

```diff
@@ -10,6 +10,8 @@
 import torch.nn as nn
 import torch.nn.functional as F
 
+from thunder.dynamo import thunderfx
```

Comment:
I'm a bit worrying about this. Nvfuser test shouldn't import thunder or depending on thunder. This feels like `import torch` in a numpy development test. 😟 

cc @jjsjann123 @kshitij12345

Reply:
a bit more context: nvfuser wheel build & test were failed, because while testing nvfuser wheels, thunder isn't installed

Reply:
Thanks for pointing this, @wujingyue had also raised concern regarding the same ([ref](https://github.com/NVIDIA/Fuser/pull/4997#discussion_r2280005068))

Plan is to move this test to thunder and then remove this one from nvFuser. I have started a PR for the same - https://github.com/Lightning-AI/lightning-thunder/pull/2450